cli/package.json

```diff
@@ -11,8 +11,8 @@
     "@apidevtools/json-schema-ref-parser": "^9.0.6",
     "@apollo/client": "^3.3.13",
     "@ffprobe-installer/ffprobe": "^1.4.1",
-    "@joystream/metadata-protobuf": "^2.5.0",
-    "@joystream/types": "^0.20.6",
+    "@joystream/metadata-protobuf": "^2.7.0",
```

Comment:
There will be apps release before ephesus and `2.6.0` is already taken, so I think `2.7.0` should be the apps and `2.8.0` - Ephesus?

Reply:
That's a sensible suggestion.